api/src/mutations/collection.rs

```diff
@@ -521,7 +521,7 @@ impl Mutation {
             return Err(Error::new("New collection must be Metaplex Certified"));
         }
 
-        if mint.compressed == true {
+        if let Some(true) = mint.compressed {
```

Comment:
Is the let required here? We aren't doing variable assignment
```suggestion
        if Some(true) == mint.compressed {
```